_posts/2015-04-18-fsharp-core-notes.md

```diff
@@ -75,7 +75,23 @@ If you are using an old-style project file, see the information further below.
 
 If using new-style .NET SDK project files, use:
 
-    <PackageReference Update="FSharp.Core" Version="4.2.3" />
+    <PackageReference Update="FSharp.Core" Version="4.5.0" />
+
+### Make your FSharp.Core references explicit
+
+Modern .NET SDK-style F# projects allow for an _implicit_ FSharp.Core reference, where the `FSharp.Core` package referenced is determined by the .NET SDK being used to compile the project.  The exact version referenced will depend on your .NET SDK, e.g. see [this table](https://github.com/fsharp/fsharp.github.io/pull/89#issuecomment-426729015).
```

Comment:
Nits:

"Allow" --> "use", as it is the default in templates. "Allow" implies this is something people can optionally turn on.

I'm not really sure there is value in linking a comment that will be out of date quite soon. If you think there is value in listing the exact version (I personally don't), then a place in the RFCs repository feels more appropriate. Or perhaps the targets files in our source code. But given that NuGet will tell you the version referenced if you encounter a warning or error, I don't see much value in tracking this.

Reply:
OK thanks, fixed. As you know I disagree with using this setting as the default in the templates